linux/dm-sworndisk/src/lib.rs

```diff
@@ -559,3 +671,77 @@ fn test_skcipher() {
     }
     assert_eq!(decrypted, plaintext);
 }
+
+fn test_rawdisk(dev_path: &'static CStr, test_size: usize) {
+    spawn(move || {
+        pr_info!("----test_rawdisk begin");
+        let rawdisk = RawDisk::open(dev_path).unwrap();
+        let nblocks = test_size / BLOCK_SIZE;
+        if rawdisk.region.end < nblocks {
+            pr_err!("raw disk is too small for test_size: {test_size}");
+            return;
+        }
+
+        let mut buf = Buf::alloc(1).unwrap();
+        pr_info!("----write begin");
```

Comment:
Is printing the common manifestation in Linux's unit tests? Since in Rust asserts are enough to represent that a test case passed or not.